plans/provision/bootc.fmf

```diff
@@ -0,0 +1,39 @@
+summary: Bootc virtual machine via testcloud
```

Comment:
This plan is currently run under the `full` packit job. Seems to be working fine, but this means that it is executed for all targets. Should we move it under the `provision` packit job? @happz, I guess that's what you meant in your [comment](https://github.com/teemtee/tmt/pull/3161#issuecomment-2468435410)? Or are we good as it is now?

Reply:
Covered by f3772f6533398dcd93b28f6c3f88bed83597e9a4.